### PR TITLE
Add Python Full Stack Guide

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: docs/python/tutorials/full-stack-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+- Install dependencies:
+
+~~~bash
+pip install -r requirements.txt
+~~~
+
+- *Optional* Create your space using Contentful Bootstrap:
+
+{: .note}
```
Comment (loudmouth) [inline docs/python/tutorials/full-stack-getting-started.md]: this too.

File: docs/python/tutorials/full-stack-getting-started.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+This guide will walk you through your first steps using Contentful within a Django application. It will provide a step-by-step guide on how to get your first entries and start using the content you create on Contentful.
+
+## Requirements
+
+- A [Heroku][7] account
+- Python 2.7+ or 3.3+
+- [Git][5] installed
+- Basic Command Line Interface, Python and Django knowledge
+
+{: .note}
```
Comment (loudmouth) [inline docs/python/tutorials/full-stack-getting-started.md]: This looks weird when viewing the rendered markdown on Github

Comment (dlitvakb) [inline docs/python/tutorials/full-stack-getting-started.md]: This is how the Kramdown renderer allows you to put custom classes (take a look at the Ruby Getting Started with the CDA for example)

**Comment by jelz** [comment]
It needs to go to `raw_docs`. Please follow https://github.com/contentful/slash-developers/blob/master/README.md#contributing

**Comment by dlitvakb** [comment]
@jelz ahhh this is new, used to go in `/docs` i'll update in a bit. I'll ping you back when I'm done.

**Comment by dlitvakb** [comment]
@jelz could you check it now? It should be in the correct directory

**Comment by jelz** [comment]
you have to run `./generate-docs.sh`, please follow https://github.com/contentful/slash-developers#contributing

**Comment by dlitvakb** [comment]
@jelz done, sorry about that, I didn't understand that i was meant to run that myself.

**Pull request merged**